src/test/fuzz/rpc.cpp

```diff
@@ -186,6 +198,84 @@ const std::vector<std::string> RPC_COMMANDS_SAFE_FOR_FUZZING{
     "waitforblockheight",
     "waitfornewblock",
 };
+// RPC commands which are safe for fuzzing.
+const std::vector<std::string> WALLET_RPC_COMMANDS_NOT_SAFE_FOR_FUZZING{
+    "importwallet",
+    "loadwallet",
+};
+// RPC commands which are safe for fuzzing.
+const std::vector<std::string> WALLET_RPC_COMMANDS_SAFE_FOR_FUZZING{
```

Comment:
I will need to move some of these over due to the rpc accessing the disk or network activity